packages/core/src/generator/index.ts

```diff
@@ -703,9 +702,13 @@ export class UnoGenerator<Theme extends object = object> {
       }
     }
 
-    // expand nested shortcuts
-    if (isString(result))
-      result = expandVariantGroup(result.trim()).split(/\s+/g)
+    if (result) {
+      result = toArray(result).map((s) => {
```

Comment:
I think using `if else` will perform better, although `toArray` will make the code more concise. But I don't want to refactor at the expense of performance.